docs/guides/self-signed-ca-certs.md

```diff
@@ -0,0 +1,50 @@
+---  
+title: Fetch Packages from Registry with Self-Signed CA Certificate  
+toc: true  
+weight: 270  
+indent: true  
+---  
+
+# Overview of Crossplane for Registry with Self-Signed CA Certificate  
+
+Crossplane uses the go-containerregistry remote pkg as the client to fetch
+package manifests and images from OCI registries. It uses the 
+http.DefaultTransport, meaning that it loads the default system certificates.
+This precludes fetching packages from private registries that may use certs
+from a self-signed CA.
+
+The default transport can be overridden with one that passes additional certs
+to the TLSConfig. This allows for self-signed certificate authorities to be
+trusted.
+
+Crossplane needs to be installed via the Helm chart with the 
+`registryCaBundleConfig.name` and `registryCaBundleConfig.key` parameters 
+defined. See [Install Crossplane](https://crossplane.io/docs/v1.6/reference/install.html#uninstalling-the-chart).
+
+## Conifgure
+
+1. Create a CA Bundle (A file containing your Root and Intermediate 
```

Comment:
Would it be worth just linking to documentation on how to do this on an external source?

Reply:
I really didn't find any official docs on it. Just blog posts. If there is official docs somewhere, I think this makes perfect sense.

Reply:
I added a note